package.json

```diff
@@ -51,6 +53,12 @@
     "eslint-plugin-react": "7.21.5",
     "eslint-plugin-react-hooks": "1.7.0",
     "eslint-plugin-testing-library": "3.3.1",
-    "prettier": "^2.2.1"
+    "husky": ">=6",
+    "lint-staged": ">=10",
+    "prettier": "^2.2.1",
+    "pretty-quick": "^3.1.0"
+  },
+  "lint-staged": {
+    "*.{tsx,ts}": "pretty-quick --staged"
```

Comment:
Is this just running prettier? Prettier is integrated into eslint via a plugin. Can we run the lint script instead?

Reply:
I don't see any mention on eslint in the pretty-quick docs - you sure it is linting?

Reply:
You're right, it's not linting. I'll change the hook to run the lint task.